topology-manager/src/main/java/org/apache/aries/rsa/topologymanager/Activator.java

```diff
@@ -98,7 +98,7 @@ private Filter exportPolicyFilter(BundleContext bc) throws InvalidSyntaxExceptio
     public void doStart(final BundleContext bc, ExportPolicy policy) {
         LOG.debug("TopologyManager: start()");
         notifier = new EndpointListenerNotifier();
-        exportExecutor = new ThreadPoolExecutor(5, 10, 50, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(), new NamedThreadFactory("TopologyManagerExporter-%d"));
+        exportExecutor = new ThreadPoolExecutor(5, 10, 50, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(), new NamedThreadFactory(TopologyManagerExport.class.getSimpleName() + "-%d"));
```

Comment:
How about making it simply:
new NamedFactory(TopologyManagerExport.class). 
This way the format string moves inside of the NamedThreadFactory and we are sure it always has the right format.

Reply:
Good suggestion, will update the PR